.conda/meta.yaml

```diff
@@ -23,7 +23,9 @@ requirements:
 
 test:
   commands:
-    - pytest
+    # We cannot call pytest directly because the source files rest in
+    # ../work_moved_gettsim_* and not in the cwd.
```

Comment:
Is `work_moved_gettsim_*` for real?

Reply:
the directory is called `work_moved_gettsim_` and then some number which changes for every build. I will clarify it with `work_moved_gettsim_[0-9]+`.

Reply:
Does not apply anymore.